index.html

```diff
@@ -36,6 +37,116 @@ <h1><b>I’m Alexandra.<br>
+        <div class="first_card">
+            <img src="Snapshoot Portfolio (9).jpg" alt="">
+            <h2>Tonic</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+
+        <div class="second_card">
+            <img src="Snapshoot Portfolio (10).jpg" alt="">
+            <h2>Multi-Post
+                   <br>
+                Stories</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+        <div class="third_card">
+            <img src="Snapshoot Portfolio (11).jpg" alt="">
+            <h2>Tonic</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+        <div class="fourth_card">
+            <img src="Snapshoot Portfolio (12).jpg" alt="">
+            <h2>Multi-Post
+                   <br>
+                Stories</h2>
+            <h3>CANOPY</h3>
+            <ul class="birth">
+                <li>Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+            <ul  class="button">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <button>See project</button>
+        </div>
+        <section class="about">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can
+                help you build a product , feature or
+                website. Look through some of my work
+                and experience! If you like what you seeand have a project you need coded,
+                dont hesitate to contact me</p>
+                <span>LETS CONNECT</span>
```

Comment:
- [ ] I kindly suggest that you make use of any header from `h2` to `h6` to surround your `LET'S CONNECT` text because it is a Header.